src/factory.js

```diff
@@ -157,6 +159,14 @@ export class Factory {
     return note;
   }
 
+  GhostNote(noteStruct) {
+    const ghostNote = new GhostNote(noteStruct);
+    if (this.stave) ghostNote.setStave(this.stave);
+    ghostNote.setContext(this.context);
+    this.renderQ.push(ghostNote);
+    return ghostNote;
```

Comment:
Totally agree. Might make the most sense to do after most elements are in the Factory. That should more-or-less coincide with the completion of the test refactoring.